cms/djangoapps/contentstore/views/videos.py

```diff
@@ -163,31 +163,31 @@ def validate_video_image(image_file):
     """
     error = None
     image_file_width, image_file_height = get_image_dimensions(image_file)
-    image_file_aspect_ratio = abs(image_file_width/float(image_file_height) - settings.VIDEO_IMAGE_ASPECT_RATIO)
+    image_file_aspect_ratio = abs(image_file_width / float(image_file_height) - settings.VIDEO_IMAGE_ASPECT_RATIO)
 
-    if not (hasattr(image_file, 'name') and hasattr(image_file, 'content_type') and hasattr(image_file, 'size')):
+    if not all(hasattr(image_file, attr) for attr in ['name', 'content_type', 'size']):
         error = _("The selected image must contain 'name', 'content_type' and 'size'.")
     elif image_file.content_type not in settings.VIDEO_IMAGE_SUPPORTED_FILE_FORMATS.values():
         error = _('The selected image contains unsupported image file type.')
```

Comment:
@mushtaqak Should we also tell what image types are supported like we are doing in frontend validation?